Changelog.md

```diff
@@ -1,5 +1,8 @@
 # Spectrometer Changelog
 
+## v2.15.22
+- The Scan URL from `fossa analyze --experimental-enable-monorepo` command was missing the branch and revisionId ([#378](https://github.com/fossas/spectrometer/pull/378))
```

Comment:
Spectrometer release notes are written like "[version x.y.z] <release log changes>" as a sentence.
Observe the previous release notes to see this pattern in action.

Given that, instead of saying "was missing", we would want to use something like this:
```suggestion
- Adds branch and revision information to the URL reported at the end of a `fossa analyze --experimental-enable-monorepo` scan. ([#378](https://github.com/fossas/spectrometer/pull/378))
```

The idea here is to say "this release now does this new or different thing" rather than "this release fixed a bug [and doesn't mention what that means to the user]".

Reply:
Makes total sense, thanks!